package.json

```diff
@@ -5,11 +5,12 @@
   "author": "JetBrains",
   "license": "SEE LICENSE IN LICENSE.txt",
   "main": "index.js",
+  "module": "index.esm.js",
```

Comment:
looks like this will break `const logosPath = require('@jetbrains/logos')` in some setups

Reply:
Let's maybe introduce a separate endpoint, something like `@jetbrains/logos/react`?

Reply:
what kind of setups? Node 13?

Reply:
webpack

Reply:
Generally, I don't think that providing completely different API in `main` vs `module` endpoints is a good idea. The intended usage of those fields is to provide the same code transpiled for different targets.